instruqt/airgap-embedded/08-deploying-a-new-version/solve-jumpbox

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# This script runs when the platform solve the challenge.
+#
+# The platform determines if the script was successful using the exit code of this
+# script. If the exit code is not 0, the script fails. 
+#
+
+echo "This is the solve script"
```

Comment:
This is probably not sufficient to make sure the check script is successful.

Reply:
Nope. It's a template I didn't delete.